task20/js/main.js

```diff
@@ -0,0 +1,93 @@
+import '../css/style.css'
+import { createElementWithClassName } from './utils/createElement'
+import { renderLoading } from './modules/loading'
+import { removeLoading } from './modules/loading'
+import { renderErrorMessage } from './modules/error'
+
+
+const parent = document.getElementById("js-parent");
+
+const getFetchData = async (endpoint) => {
+    const response = await fetch(endpoint);
+
+    if (!response.ok){
+        console.error(`$(response:status):$(response:statusText)`);
+        renderErrorMessage(parent, "問題が発生し表示することができません。");
+    }
```

Comment:
MUST
I intentionally tried to cause a 503 error and the console was not displaying correctly.
This URL was used for verification.
`https://httpstat.us/503`

![Screenshot at Nov 06 22-25-31](https://user-images.githubusercontent.com/67426438/200173374-3e7cce0f-fef4-41e1-9a3b-c224ac5e9dc8.png)

Reply:
Sorry. I wrote it wrong and it didn't show the error. I fixed it.
[be5fbe7](https://github.com/seeeen044/JavaScript_handson/pull/21/commits/be5fbe7d3a3c2c5f21a4caa2f5ec642a700f65cb)